### PR TITLE
v0.1.0 Release

### PR DESCRIPTION
**Pull request by greyscaled**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+ * @file Exports Configuration for ESLint
+ *
+ * @author Vapurrmaid <vapurrmaid@gmail.com>
+ *
+ * @license Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+ * This code can be freely viewed and forked as per the GitHub
+ * terms of service. However, it may not be modified or distributed.
+ * The members of {@link https://github.com/KevinMarley} hold the sole rights
+ * for contribution and modification.
+ */
```
Comment (greyscaled) [inline .eslintrc.js:11]: 👍

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+#
+# Author: Vapurrmaid <vapurrmaid@gmail.com>
+#
+# Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+# This code can be freely viewed and forked as per the GitHub
+# terms of service. However, it may not be modified or distributed.
+# The members of {@link https://github.com/KevinMarley} hold the sole rights
+# for contribution and modification.
+####
+
```
Comment (greyscaled) [inline .gitignore:12]: 👍

File: babel.config.js [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+ * project, including the `node_modules/` directory.
+ *
+ * @author Vapurrmaid <vapurrmaid@gmail.com>
+ *
+ * @license Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+ * This code can be freely viewed and forked as per the GitHub
+ * terms of service. However, it may not be modified or distributed.
+ * The members of {@link https://github.com/KevinMarley} hold the sole rights
+ * for contribution and modification.
+ */
```
Comment (greyscaled) [inline babel.config.js:14]: 👍

File: bin/createcname.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo -n "kevinmarley.ca" > dist/CNAME
```
Comment (greyscaled) [inline bin/createcname.sh:2]: 👍

File: package.json [hunk 5]
```diff
@@ -1,12 +1,17 @@
 {
   "name": "kmcc18",
-  "version": "0.0.0",
+  "version": "0.1.0",
```
Comment (greyscaled) [inline package.json:3]: 👍

File: package.json [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
-  "homepage": "",
-  "bugs": "",
+  "keywords": [
+    "politics",
+    "hamont",
+    "hamilton",
+    "city council"
+  ],
+  "homepage": "https://kevinmarley.ca",
+  "bugs": "https://github.com/KevinMarley/KevinMarley.github.io/issues",
```
Comment (greyscaled) [inline package.json:12]: 👍

File: package.json [hunk 7]
```diff
@@ -1,12 +1,17 @@
...
   "license": "UNLICENSED",
-  "people": {
+  "author": {
```
Comment (greyscaled) [inline package.json:14]: 👍

File: package.json [hunk 8]
```diff
@@ -17,11 +22,11 @@
     }
   ],
   "browser": "src/main.js",
-  "repository": "",
+  "repository": "https://github.com/KevinMarley/KevinMarley.github.io",
```
Comment (greyscaled) [inline package.json:25]: 👍

File: package.json [hunk 9]
```diff
@@ -17,11 +22,11 @@
...
   "scripts": {
     "serve": "vue-cli-service serve",
     "copyreadme": "cp README.md dist/README.md",
-    "createcname": "createcname.sh",
+    "createcname": "bash ./bin/createcname.sh",
```
Comment (greyscaled) [inline package.json:29]: 👍

File: package.json [hunk 10]
```diff
@@ -50,5 +55,8 @@
     "sass-loader": "^7.0.1",
     "vue-template-compiler": "^2.5.17"
   },
+  "engines": {
+    "node": ">=8.11.0"
+  },
```
Comment (greyscaled) [inline package.json:60]: 👍

File: postcss.config.js [hunk 11]
```diff
@@ -1,5 +1,21 @@
...
+ * CSS and modifies it for production. PostCSS is applied via webpack
+ *
+ * @author Vapurrmaid <vapurrmaid@gmail.com>
+ *
+ * @license Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+ * This code can be freely viewed and forked as per the GitHub
+ * terms of service. However, it may not be modified or distributed.
+ * The members of {@link https://github.com/KevinMarley} hold the sole rights
+ * for contribution and modification.
+ */
```
Comment (greyscaled) [inline postcss.config.js:12]: 👍

File: postcss.config.js [hunk 12]
```diff
@@ -1,5 +1,21 @@
...
 module.exports = {
   plugins: {
+    /**
+     * Automatically adds vendor prefixes as per the data
+     * at {@link https://caniuse.com|Can I use}
+     */
```
Comment (greyscaled) [inline postcss.config.js:18]: 👍

File: src/App.vue [hunk 13]
```diff
@@ -1,19 +1,18 @@
...
 
 Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
 
 This code can be freely viewed and forked as per the GitHub
 terms of service. However, it may not be modified or distributed.
 The members of {@link https://github.com/KevinMarley} hold the sole rights
 for contribution and modification.
-
- -->
+-->
```
Comment (greyscaled) [inline src/App.vue:15]: 👍

File: src/components/Drawer.vue [hunk 14]
```diff
@@ -1,10 +1,21 @@
...
+
+Author: Vapurrmaid <vapurrmaid@gmail.com>
+
+Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+
+This code can be freely viewed and forked as per the GitHub
+terms of service. However, it may not be modified or distributed.
+The members of {@link https://github.com/KevinMarley} hold the sole rights
+for contribution and modification.
+-->
```
Comment (greyscaled) [inline src/components/Drawer.vue:14]: 👍

File: src/components/Election.vue [hunk 15]
```diff
@@ -60,6 +74,9 @@
 import Card from '@/components/common/Card'
 import Section from '@/components/common/Section'
 
+/**
+ * Exports a set of cards detailing upcoming election information
+ */
```
Comment (greyscaled) [inline src/components/Election.vue:79]: 👍

**Comment by greyscaled** [comment]
@ChockIZ  - can you read/check over content of [readme (here)](https://github.com/KevinMarley/KevinMarley.github.io/blob/90ead49f22aac5fdeb77ddc568d9de64cab3206d/README.md) then submit an approval or review with comment for change

File: src/components/Footer.vue [hunk 16]
```diff
@@ -84,6 +99,9 @@
 </template>
 
 <script>
+/**
+ * Exports a component representing a site-wide Footer.
+ */
```
Comment (greyscaled) [inline src/components/Footer.vue:104]: 👍

File: src/components/Header.vue [hunk 17]
```diff
@@ -26,6 +40,9 @@
 </template>
 
 <script>
+/**
+ * Exports component detailing a site header
+ */
```
Comment (greyscaled) [inline src/components/Header.vue:45]: 👍

File: src/components/Meet.vue [hunk 18]
```diff
@@ -40,14 +54,16 @@
...
     </div>
   </Section>
 </template>
 
 <script>
 import Section from '@/components/common/Section'
 
+/**
+ * Exports a component containing articles and pictures
+ */
```
Comment (greyscaled) [inline src/components/Meet.vue:66]: 👍

File: src/components/Nav.vue [hunk 19]
```diff
@@ -16,13 +31,20 @@
...
 <script>
 import { mapActions, mapGetters } from 'vuex'
 import Drawer from '@/components/Drawer'
 
+/**
+ * Exports a navigation menu button that toggles a drawer
+ *
+ * @vue-computed {boolean} isNavDrawerOpen
+ * @property {function} toggleNavDrawer
+ */
```
Comment (greyscaled) [inline src/components/Nav.vue:47]: 👍

File: src/components/Priorities.vue [hunk 20]
```diff
@@ -122,6 +132,10 @@
 import CardSlider from '@/components/common/CardSlider'
 import Section from '@/components/common/Section'
 
+/**
+ * Exports Cards and an article detailing various
+ * campaign priorities.
+ */
```
Comment (greyscaled) [inline src/components/Priorities.vue:138]: 👍

File: src/components/common/Card.vue [hunk 21]
```diff
@@ -1,10 +1,27 @@
...
+for contribution and modification.
+-->
 <template>
   <div class="card">
     <slot></slot>
   </div>
 </template>
 
 <script>
+/** Card component */
```
Comment (greyscaled) [inline src/components/common/Card.vue:24]: 👍

File: src/components/common/CardSlider.vue [hunk 22]
```diff
@@ -1,3 +1,17 @@
...
+
+Author: Vapurrmaid <vapurrmaid@gmail.com>
+
+Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+
+This code can be freely viewed and forked as per the GitHub
+terms of service. However, it may not be modified or distributed.
+The members of {@link https://github.com/KevinMarley} hold the sole rights
+for contribution and modification.
+-->
```
Comment (greyscaled) [inline src/components/common/CardSlider.vue:14]: 👍

File: src/components/common/Section.vue [hunk 23]
```diff
@@ -1,7 +1,18 @@
...
+
+Author: Vapurrmaid <vapurrmaid@gmail.com>
+
+Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+
+This code can be freely viewed and forked as per the GitHub
+terms of service. However, it may not be modified or distributed.
+The members of {@link https://github.com/KevinMarley} hold the sole rights
+for contribution and modification.
+-->
```
Comment (greyscaled) [inline src/components/common/Section.vue:15]: 👍

File: src/main.js [hunk 24]
```diff
@@ -1,11 +1,9 @@
...
- *
- * @license
  *
- * Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+ * @author Vapurrmaid <vapurrmaid@gmail.com>
  *
+ * @license Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
  * This code can be freely viewed and forked as per the GitHub
  * terms of service. However, it may not be modified or distributed.
  * The members of {@link https://github.com/KevinMarley} hold the sole rights
```
Comment (greyscaled) [inline src/main.js:9]: 👍

File: src/sass/abstracts/_animations.scss [hunk 25]
```diff
@@ -1,7 +1,8 @@
 /*
-Globally available Animations definitions and classes.
+File: _animations.scss
+Description: Globally available Animations definitions and classes.
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/abstracts/_animations.scss:5]: 👍

File: src/sass/abstracts/_mixins.scss [hunk 26]
```diff
@@ -1,7 +1,8 @@
 /*
-Globally available Style Mixins. Uses variables.scss.
+File: _mixins.scss
+Description: Globally available Style Mixins. Uses variables.scss.
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/abstracts/_mixins.scss:5]: 👍

File: src/sass/abstracts/_variables.scss [hunk 27]
```diff
@@ -1,7 +1,8 @@
 /*
-Globally available Variables.
+File: _variables.scss
+Description: Globally available Variables.
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/abstracts/_variables.scss:5]: 👍

File: src/sass/base/_base.scss [hunk 28]
```diff
@@ -1,7 +1,8 @@
 /*
-Global resets.
+File: _base.scss
+Description: Global resets.
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/base/_base.scss:5]: 👍

File: src/sass/base/_typography.scss [hunk 29]
```diff
@@ -1,7 +1,9 @@
 /*
-Globally available Font Definitions.
+File: _typography.scss
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Description: Globally available Font Definitions.
+
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/base/_typography.scss:6]: 👍

File: src/sass/base/_utils.scss [hunk 30]
```diff
@@ -1,7 +1,9 @@
 /*
-Globally available collection of Utilities.
+File: _utils.scss
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Description: Globally available collection of Utilities.
+
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/base/_utils.scss:6]: 👍

File: src/sass/main.scss [hunk 31]
```diff
@@ -1,8 +1,10 @@
 /*
-Entry point for Sass Compilation. To be imported
+File: main.scss
+
+Description: Entry point for Sass Compilation. To be imported
 in the root of the Vue application.
 
-Author Vapurrmaid <vapurrmaid@gmail.com>
+Author: Vapurrmaid <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/sass/main.scss:7]: 👍

File: src/store/index.js [hunk 32]
```diff
@@ -1,11 +1,9 @@
 /**
  * @file Configures Vuex Store.
- * @author Vapurrmaid <vapurrmaid@gmail.com>
- *
- * @license
  *
- * Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+ * @author Vapurrmaid <vapurrmaid@gmail.com>
  *
+ * @license Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/store/index.js:6]: 👍

File: src/store/modules/application.js [hunk 33]
```diff
@@ -1,12 +1,10 @@
...
  * @file Configures a Vuex Store module called 'application'. This
  * module is intended to be used globally across the entire Vue application.
- * @author Vapurrmaid <vapurrmaid@gmail.com>
- *
- * @license
  *
- * Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
+ * @author Vapurrmaid <vapurrmaid@gmail.com>
  *
+ * @license Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
```
Comment (greyscaled) [inline src/store/modules/application.js:7]: 👍

File: src/views/Home.vue [hunk 34]
```diff
@@ -1,17 +1,17 @@
...
 
 Copyright (C) 2018 kevinmarley.ca <vapurrmaid@gmail.com>
 
 This code can be freely viewed and forked as per the GitHub
 terms of service. However, it may not be modified or distributed.
 The members of {@link https://github.com/KevinMarley} hold the sole rights
 for contribution and modification.
-
- -->
+-->
```
Comment (greyscaled) [inline src/views/Home.vue:14]: 👍

File: tests/unit/.eslintrc.js [hunk 35]
```diff
@@ -1,8 +1,20 @@
...
+  /** Mocha test environment allows use of `describe`, `it` etc. */
   env: {
     mocha: true
   },
   rules: {
     'import/no-extraneous-dependencies': 'off'
   }
-}
\ No newline at end of file
+}
```
Comment (greyscaled) [inline tests/unit/.eslintrc.js:20]: 👍

File: tests/helpers/index.js [hunk 36]
```diff
@@ -1,4 +1,44 @@
...
 const mockAction = () => {
+  /**
+   * Mocked Vuex Action
+   *
+   * @typedef mockAction
+   * @type {Object}
+   * @property {number} called
+   * @property {function} getCalled
+   * @property {function} action
+   */
```
Comment (greyscaled) [inline tests/helpers/index.js:41]: 👍

File: tests/unit/Card.spec.js [hunk 37]
```diff
@@ -1,14 +1,15 @@
 /* eslint-env mocha */
 
-import { mount } from '@vue/test-utils'
+import { shallowMount } from '@vue/test-utils'
```
Comment (greyscaled) [inline tests/unit/Card.spec.js:3]: 👍

File: tests/unit/Card.spec.js [hunk 38]
```diff
@@ -1,14 +1,15 @@
...
 import { expect } from 'chai'
 import Card from '@/components/common/Card.vue'
 
 describe('App', function () {
   beforeEach(function () {
-    this.wrapper = mount(Card)
+    this.wrapper = shallowMount(Card)
   })
 
+  // smoke test
```
Comment (greyscaled) [inline tests/unit/Card.spec.js:12]: 👍

File: tests/unit/CardSlider.spec.js [hunk 39]
```diff
@@ -19,25 +20,29 @@ describe('CardSlider', function () {
...
   })
 
+  // check alt prop
   it('adds the alt text to the provided image', function () {
     const img = this.wrapper.find('img')
     expect(img.attributes().alt)
       .to.equal(TESTING_ALT)
   })
 
+  // check title prop
```
Comment (greyscaled) [inline tests/unit/CardSlider.spec.js:45]: 👍

File: tests/unit/Drawer.spec.js [hunk 40]
```diff
@@ -41,11 +42,13 @@ describe('Drawer', function () {
     expect(this.wrapper.contains('nav.nav'))
   })
 
+  // smoke test for each set of nav links
   it('contains a list of links and list external links', function () {
     expect(this.wrapper.contains('nav.nav ul.nav__list'))
     expect(this.wrapper.contains('nav.nav ul.nav__list--external'))
   })
 
+  // ensures clicking a link calls the appropriate store action
```
Comment (greyscaled) [inline tests/unit/Drawer.spec.js:51]: 👍

File: tests/unit/Election.spec.js [hunk 41]
```diff
@@ -9,6 +9,7 @@ describe('Election', function () {
     this.wrapper = shallowMount(Election)
   })
 
+  // smoke test
```
Comment (greyscaled) [inline tests/unit/Election.spec.js:12]: 👍

File: tests/unit/Footer.spec.js [hunk 42]
```diff
@@ -17,6 +18,7 @@ describe('Footer', function () {
       .and.to.have.lengthOf(2)
   })
 
+  // smoke test for copyright
```
Comment (greyscaled) [inline tests/unit/Footer.spec.js:21]: 👍

File: tests/unit/Header.spec.js [hunk 43]
```diff
@@ -9,18 +9,21 @@ describe('Header', function () {
...
       .and.to.have.lengthOf(1)
   })
 
+  // smoke test for <h1>
   it('renders a primary heading h1', function () {
     expect(this.wrapper.find('header h1.heading-primary').exists())
       .to.equal(true)
   })
 
+  // smoke test for <h2>
```
Comment (greyscaled) [inline tests/unit/Header.spec.js:26]: 👍

File: tests/unit/Meet.spec.js [hunk 44]
```diff
@@ -9,13 +9,15 @@ describe('Meet', function () {
...
 
+  // smoke test
   it('renders a root div with class about', function () {
     expect(this.wrapper.find('div').classes())
       .to.be.an('array')
       .that.includes('about')
       .and.to.have.lengthOf(1)
   })
 
+  // smoke test for article
```
Comment (greyscaled) [inline tests/unit/Meet.spec.js:20]: 👍

File: tests/unit/Nav.spec.js [hunk 45]
```diff
@@ -55,6 +58,7 @@ describe('Nav', function () {
       .and.not.includes('navigation__hamburger--open')
   })
 
+  // check reacts to store getter - open case
```
Comment (greyscaled) [inline tests/unit/Nav.spec.js:61]: 👍

File: tests/unit/Priorities.spec.js [hunk 46]
```diff
@@ -9,23 +9,27 @@ describe('Priorities', function () {
...
       .to.equal('Priorities')
   })
 
+  // smoke test for nested <h4>
   it('renders a heading for Ward 12 Priorities', function () {
     expect(this.wrapper.find('.ward12__container .row:first-child h4').text())
       .to.equal('Ward 12 Priorities')
   })
 
+  // smoke test for nested <h4>
```
Comment (greyscaled) [inline tests/unit/Priorities.spec.js:32]: 👍

File: tests/unit/Section.spec.js [hunk 47]
```diff
@@ -1,22 +1,24 @@
 /* eslint-env mocha */
 
-import { mount } from '@vue/test-utils'
+import { shallowMount } from '@vue/test-utils'
```
Comment (greyscaled) [inline tests/unit/Section.spec.js:3]: 👍

File: tests/unit/Section.spec.js [hunk 48]
```diff
@@ -1,22 +1,24 @@
...
       }
     })
   })
 
+  // smoke test
   it('renders a root section with class section', function () {
     expect(this.wrapper.contains('section.section'))
   })
 
+  // check can accept child nodes
```
Comment (greyscaled) [inline tests/unit/Section.spec.js:21]: 👍

**Review by greyscaled — COMMENTED** [review]
- fix `router.js` in #29  then should be good to merge

**Review by ChockIZ — APPROVED** [review]
Seems good to me, not sure if I was viewing it right but from what I saw looked good.

**Comment by greyscaled** [comment]
@ChockIZ just the README ya

**Comment by greyscaled** [comment]
Working on a rebase/pick to show 2 commits, then merge

**Comment by greyscaled** [comment]
**note** Since review passed, force pushed rebase - ready for merge

**Pull request merged**